tests/juju-extra-bootstrap-args/concierge.yaml

```
@@ -0,0 +1,14 @@
+juju:
+  channel: 4.0/stable
+  model-defaults:
+    test-mode: "true"
+  extra-bootstrap-args: --config idle-connection-timeout=90s
```

```suggestion
  extra-bootstrap-args: --config model-logs-size=10M
```

`idle-connection-timeout` is brand new in Juju, so not available in the version we're testing against.